config/locales/client.en.yml

```diff
@@ -4077,7 +4077,7 @@ en:
         one: "%{count} new"
         other: "%{count} new"
       toggle_section: "toggle section"
-      more_count: "%{count} more..."
+      more: "more..."
```

Comment:
nit: I think we should capitalize this `More...`

Reply:
Done :)